### PR TITLE
Chown troubleshooting tip

### PR DESCRIPTION
**Pull request by johnpyp**
- Adds a line in Sonarr section about possibly needing to chown your root directory to fix permission issues
- Fixed minor typos

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -562,11 +562,13 @@ In `Connect` tab, we'll configure Sonarr to send notifications to Plex when a ne
 
 #### Give it a try
 
-Let's add a serie !
+Let's add a series !
```
Comment (sebgl) [inline README.md]: Hah, that's my poor english 😄 
Would it make more sense to call it "tv show"?

File: README.md [hunk 2]
```diff
@@ -562,11 +562,13 @@ In `Connect` tab, we'll configure Sonarr to send notifications to Plex when a ne
...
 
 ![Adding a serie](img/sonarr_add.png)
 
-Enter the serie name, then you can choose a few things:
+*Note: You may need to `chown -R $USER:$USER /path/to/root/directory` so Sonarr and the rest of the apps have the proper permissions to modify and move around files*
```
Comment (sebgl) [inline README.md]: Nit-picking: maybe add a final dot `.` to this line?

File: README.md (hunk 1 above)
Comment (johnpyp) [reply README.md]: Series and tv show are interchangeable, so in this case I think series is fine. However, when it comes to folder naming, it seems like `/tv` is the agreed upon term; this isn't an issue here though.

File: README.md (hunk 2 above)
Comment (johnpyp) [reply README.md]: Sure

**Pull request merged**